firstvoices/backend/management/commands/copy_site.py

```diff
@@ -263,32 +264,48 @@ def copy_videos_and_return_map(source_site, target_site, set_modified_date, logg
 
 def copy_galleries(source_site, target_site, image_map, set_modified_date, logger):
     galleries = list(Gallery.objects.filter(site=source_site))
+
+    shared_media_sites_ids = list(
+        SiteFeature.objects.filter(key="shared_media").values_list("site", flat=True)
```

Comment:
This flag can be enabled or disabled, so we need one extra filter in here to get only the enabled ones